traceroute/traceroute.c

```diff
@@ -784,9 +784,14 @@ static void print_probe (probe *pb) {
 }
 
 
-static void print_end (void) {
+static void print_end (int result) {
 
-	printf ("\n");
+	if (result)
+		printf("\nTrace successful\n");
+	else
+		printf("\nTrace failed\n");
```

Comment:
The Jira comment says to also: 
```
After 30 hops - Trace Failed
After reaching destination beofre 30 hops - Trace Successful
Abort with ctrl+c - Trace Aborted```
https://drivenets.atlassian.net/browse/SW-57507?focusedCommentId=290957

Reply:
`Abort with ctrl+c - Trace Aborted` - this will be handled from CLI